src/components/Organisms/WYMDCarousel/WYMDCarousel.js

```diff
@@ -0,0 +1,234 @@
+    if (typeof window !== 'undefined') window.onresize = screenResize;
+  }, [screenResize]);
+
+  if (theseItems && totalSlides === null) {
+    // Reflects our two dummy/bookend slides for non-mobile/tablet views:
+    setTotalSlides(isMobileOrTablet ? theseItems.length : theseItems.length + 2);
+  }
+
+  return (
+    <CarouselWrapper
+      className="CarouselWrapper"
+      id={thisID}
+    >
+
+      <Heading tag="p" weight="bold">
+        { data.heading}
+      </Heading>
+
+      <PeopleHelpedText tag="h1" family="Anton" uppercase weight="normal" color="red">
+        { data.peopleHelpedText}
+      </PeopleHelpedText>
+
+      <Including tag="p">
+        including...
+      </Including>
+
+      {theseItems && (
+      <CarouselProvider
+        naturalSlideWidth={50}
+        naturalSlideHeight={200}
+        totalSlides={totalSlides}
+        isPlaying={autoPlay}
+        interval={5000}
+        visibleSlides={visibleSlides}
+        infinite
+      >
+        <Slider classNameAnimation="wymd-carousel">
+
+          {/* Dummy slide for our desired non-mobile layout and functionality */}
+          {isMobileOrTablet === false && (
+          <Slide index={0} key={0} />
+          )}
+
+          {Object.keys(theseItems).map((key, index) => {
+            // Reflect that initial dummy/bookend slide shown on non-mobile/tablet views:
+            const thisOffsetIndex = index + (isMobileOrTablet ? 0 : 1);
+
+            return (
+              // Calculate the index offset accordingly to reflect the number of slides,
+              // but use the REAL index when determining if its the last REAL slide
+              <Slide
+                index={thisOffsetIndex}
+                className={index === (theseItems.length - 1) && 'last-slide'}
+                key={thisOffsetIndex}
+              >
+
+                <ImageWrapper className="image-wrapper">
+                  <img src={theseItems[key].image.file.url} alt={theseItems[key].copy} />
+                </ImageWrapper>
+
+                <AmountWrapper>
+                  <Text tag="h1" family="Anton" uppercase weight="normal" size="super">
+                    {theseItems[key].amount}
+                  </Text>
+                </AmountWrapper>
+
+                <CopyWrapper>
+                  <Text tag="p" size="l">
+                    {theseItems[key].copy}
+                  </Text>
+                </CopyWrapper>
+              </Slide>
+            );
+          })}
+
+          {/* Dummy slide for our desired non-mobile layout and functionality */}
+          {isMobileOrTablet === false && (
+          <Slide index={theseItems.length + 1} key="bookend-last" />
+          )}
+
+        </Slider>
+        <ButtonBack>Back</ButtonBack>
+        <ButtonNext>Next</ButtonNext>
+      </CarouselProvider>
+      )}
+
+    </CarouselWrapper>
+  );
+};
+
+WYMDCarousel.propTypes = {
+  data: PropTypes.shape({
+    // Required 'node' fields:
```

Comment:
Rather than eat up yet another content type to embed here as a 'Node' or whatever, Curtis and I decided that it just made more sense to have everything self-contained.

Just the first four 'nodes' (made up of amount, copy and image fields) are required in the CMS, everything else is optional.